MaxText/experimental/rl/grpo_trainer.py

```diff
@@ -639,6 +639,7 @@ def generation_worker_fn(
   generation_thread.start()
 
   try:
+    last_step_completion = datetime.datetime.now()
```

Comment:
Does this + the timing below work properly without `jax.block_until_ready(...)`?